framework/src/media/FocusManager.cpp

```diff
@@ -30,6 +30,11 @@ bool FocusManager::FocusRequester::hasSameId(std::shared_ptr<FocusRequest> focus
 	return mId == focusRequest->getStreamInfo()->id;
 }
 
+bool FocusManager::FocusRequester::pred(const FocusManager::FocusRequester a, const FocusManager::FocusRequester b)
```

Comment:
something including 'compare' will be better

Reply:
The Pred function returns the predecessor of an ordinal value.
I found that qsort in <stdlib.h> and sort in <algorithm> get `compare` parameter.
So, I decide to name as `compare`.